microsoft-edge/toc.yml

```
@@ -345,6 +345,10 @@
                   href: devtools-guide-chromium/progressive-web-apps/index.md
                   displayName: Application tool
 
+                - name: Test protocol handlers for PWAs
```

```suggestion
                - name: Test Progressive Web App (PWA) protocol handling
```
I think Patrick is saying to make this change if top-of-article keeps this wording.